src/main/java/de/baleipzig/iris/ui/presenter/training/TrainingPresenter.java

```diff
@@ -19,6 +19,8 @@
     private TrainingViewModel model = new TrainingViewModel();
     private final DozerBeanMapper dozerBeanMapper = new DozerBeanMapper();
 
+    private INeuralNetTrainer<BufferedImage, Integer> trainer;
```

Comment:
Vorhin nicht aufgefallen. können wir uns den nicht injecten lassen. Konfigurationsklasse analog zu Evo und dann als final member in den service. würde den Presenter auch wieder entschlacken. Kannst aber zur Not erstmal so lassen und wir schauen gemeinsam nochmal